pages/api/system/reload.js

```diff
@@ -210,6 +210,14 @@ export default async function handler(req, res) {
                   end: event.date.end,
                 },
                 url: event.url,
+                location: event.location? {
+                  road:event.location.road?event.location.road: "",
+                  city:event.location.city?event.location.city: "",
+                  state:event.location.state?event.location.state: "",
+                  country:event.location.country?event.location.country: "",
+                  lat:event.location.lat?event.location.lat: "",
+                  lon:event.location.lon?event.location.lon: ""
+                }: undefined,
```

Comment:
Please check your VScode for the Prettier plugin, so your code will be formatted correctly

Reply:
Done.